libkbfs/kbfs_ops.go

```diff
@@ -476,10 +476,27 @@ func (fs *KBFSOpsStandard) createAndStoreTlfIDIfNeeded(
 	return fs.resetTlfID(ctx, h)
 }
 
+func (fs *KBFSOpsStandard) transformReadError(
+	ctx context.Context, h *TlfHandle, err error) error {
+	if errors.Cause(err) != context.DeadlineExceeded {
+		return err
+	}
+
+	if fs.config.IsSyncedTlf(h.tlfID) {
+		fs.log.CWarningf(ctx, "Got a read timeout on a synced TLF: %+v", err)
+		return err
+	}
+
+	// For unsynced TLFs, return a specific error to let the system
+	// know to show a sync recommendation.
+	return errors.WithStack(OfflineUnsyncedError{h})
```

Comment:
These could stack up with the fbo transform, right? I guess it's harmless though since we'll just have multiple log lines.

Reply:
Not sure what you mean by "stack up", but yeah one `OfflineUnsyncedError` could be replaced by another one here, for no real reason.  I'll add a quick check to avoid that.